### DISCUSSION TITLE
Active leaves of doorways with two leaves

### DISCUSSION THREAD

**Discussion by greeshmaunni1990** [IfcOpenShell/IfcOpenShell#2326] [Q&A]
Hi,
I'm utilizing IFC files to construct a manual using Python, and one of the sections of the manual is about figuring out which leaves of a double swinging door are active. Could someone help me with this?
Thanks

**Comment by Moult** [comment]
What do you mean by "active", sorry?

**Reply by greeshmaunni1990** [reply]
Hi,
In case of double doors we have two panels. I need to check if atleast one of those leaves are swinging(while the other remains stationary)
